### PR TITLE
Fix Toolbar Stories

### PR DESCRIPTION
**Pull request by cmbirk**

### CODE REVIEW

**Comment by Blakidarkness** [comment]
There is a bug when the tool it's disabled, it changes the styling but the handler stills working.
Seems that the only tool that works correctly its `can Reset`

**Comment by cmbirk** [comment]
Thank you @Blakidarkness I've created an issue to address this here as the current GH actions are broken and this is the fix for that

https://github.com/CrossroadsCX/datatable/issues/15

File: src/DataTable/DataTable.stories.tsx [hunk 1]
```diff
@@ -29,19 +29,19 @@ const defaultSale = {
 
 const originalData: Sale[] = [
   {
-    date: format(new Date(), 'MM-dd-yyyy'),
+    date: format(new Date('08-27-2021'), 'MM-dd-yyyy'),
```
Comment (Emmaalara) [inline src/DataTable/DataTable.stories.tsx:32]: I think that when we declare default values is a good practice to replace them with a constant.

```suggestion
    const date = '08-27-2021';
    date: format(new Date(date), 'MM-dd-yyyy'),
```

**Review by Blakidarkness — APPROVED** [review]
Approved, bug fix addressed to: #15

**Review by Emmaalara — COMMENTED** [review]
I think that when we declare default values is a good practice to replace them with a constant.

**Pull request merged**